index.js

```diff
@@ -94,7 +94,16 @@ client.songRecs = new Enmap({name:'songs'});//create new enmap for song recommen
 client.songRecs.fetchEverything();
 
 
-
+configArray = ["token","prefix","weather_token","stock_token","news_token","d","owner","nasa_token","crypto_token"];
+const jsonConfig = require('./config.json');
+for(let i = 0; i < configArray.length;i++){
```

Comment:
a foreach over the configarray would make it so you don't have to deal with index variables at all, maybe make it a bit more readable as well

Reply:
like configArray.foreach(fxn)?

Reply:
correct